src/editor.js

```diff
@@ -1502,6 +1502,29 @@ class Editor {
         this.renderer.removeGhostText();
     }
 
+    /**
+     * Set "widgetManager" in editor.session
+     * @returns void
+     */
+    setWidgetManager() {
```

Comment:
calling this `setWidgetManager` might lead to the assumption that it accepts a parameter which gets set, maybe something like `attachWidgetManager` makes sense?